unity-renderer/Assets/DCLServices/CameraReelService/CameraReelNetworkStorageService.cs

```diff
@@ -35,10 +37,15 @@ public async UniTask<CameraReelStorageStatus> DeleteScreenshot(string uuid, Canc
             return new CameraReelStorageStatus(response.currentImages, response.maxImages);
         }
 
-        public async UniTask<(CameraReelResponse, CameraReelStorageStatus)> UploadScreenshot(Texture2D image, ScreenshotMetadata metadata, CancellationToken ct = default)
+        public async UniTask<CameraReelStorageStatus> UploadScreenshot(Texture2D image, ScreenshotMetadata metadata, CancellationToken ct = default)
```

Comment:
In my opinion, the upload operation should return the result of the picture, which is `CameraReelResponse`. Triggering the result through an event doesnt mean that the operation should change its return values

Reply:
Agree! You can resolve it